core/datasets.go

```diff
@@ -576,32 +576,31 @@ func (r *DatasetRequests) Remove(p *repo.DatasetRef, ok *bool) (err error) {
 	return nil
 }
 
-// StructuredDataParams defines parameters for retrieving
-// structured data (which is the kind of data datasets contain)
-type StructuredDataParams struct {
+// LookupParams defines parameters for looking up the body of a dataset
+type LookupParams struct {
 	Format        dataset.DataFormat
 	FormatConfig  dataset.FormatConfig
 	Path          string
 	Limit, Offset int
 	All           bool
 }
 
-// StructuredData combines data with it's hashed path
-type StructuredData struct {
+// LookupResult combines data with it's hashed path
+type LookupResult struct {
```

Comment:
This is great for now. In the future I think we should:
* land a change to `dataset.DatasetPod` that renames "Data" to "Body"
* remove `LookupResult` & replace with a `repo.DatasetRef` that has `ref.Dataset.Body` populated